docker/lr-cartographer/install_R_packages.R

```diff
@@ -0,0 +1,42 @@
+options(warn = 2)     # treat warnings as errors, otherwise script can fail silently if a package fails to install
+
+InstallPackageFromArchive = function(packageName, packageURL) {
+		# make sure to use http not https as this will give an "unsupported URL scheme" error
+		if (!(packageName %in% rownames(installed.packages()))) {
+				install.packages(packageURL, repos = NULL, type = "source", clean = TRUE)
+		}
+}
+
+dependencies = c("qqman", 
+								 "digest", "gtable", "MASS", "plyr", "reshape2", "scales", "tibble", "lazyeval", # for ggplot2
+								 "ggplot2")
+#repos <- c("http://cran.mtu.edu")
```

Comment:
Actually, I want to leave these in so that if we ever want to go back and change the sources we can easily do so (without looking up what to change them to).